local/php/envs.go

```diff
@@ -29,22 +29,38 @@ import (
 	"github.com/symfony-cli/symfony-cli/envs"
 )
 
-func (p *Server) generateEnv(req *http.Request) map[string]string {
-	scriptName := p.passthru
-	https := ""
-	if req.TLS != nil {
-		https = "On"
-	}
-
-	pathInfo := req.URL.Path
+func (p *Server) resolveScriptName(pathInfo string) (string, string) {
 	if pos := strings.Index(strings.ToLower(pathInfo), ".php"); pos != -1 {
 		file := pathInfo[:pos+4]
 		if _, err := os.Stat(filepath.Join(p.documentRoot, file)); err == nil {
-			scriptName = file
-			pathInfo = pathInfo[pos+4:]
+			return file, pathInfo[pos+4:]
+		}
+	}
+
+	paths := strings.Split(strings.Trim(pathInfo, "/"), "/")
```

Comment:
Using `strings.Split` (or `Trim` /`LeftTrim` as far as I can tell) will actually lead to potential directory traversal:
the heading "/" being removed it becomes possible for one to request an URL that will have ".." components interpreted from the document root and leaving the possibility to one to execute a script outside of the document root.

Reply:
I am gonna test it, not sure how golang behaves with ../. I guessed if it's an issue it would be an issue already before with direct PHP file in url.